public/test/mock-api/scenarios.ts

```diff
@@ -0,0 +1,25 @@
+import { HttpHandler } from 'msw';
+
+import { SILENCES_SCENARIOS } from './scenarios/alerting/silences';
+
+const ALL_SCENARIOS = {
+  ...SILENCES_SCENARIOS,
+};
+
```

Comment:
I wanted to do something with an array of objects with scenarios, and combining them and checking we didn't have duplicates - but I kept ending up losing type safety. Having the `Scenario` type is v useful